apps/meteor/client/sidebar/RoomMenu.js

```diff
@@ -139,6 +141,24 @@ const RoomMenu = ({ rid, unread, threadUnread, alert, roomOpen, type, cl, name =
 		}
 	});
 
+	const handleThreadsToBeRead = useMutableCallback(async () => {
+		try {
+			if (threadUnread) {
+				const { threads } = await getThreadsList({
+					rid,
+					type: 'unread',
+				});
+				const promises = [];
+				for (const thread of threads) {
+					promises.push(readThreads(thread?._id));
+				}
```

Comment:
```suggestion
				const promises = threads.map(thread => readThreads(thread?._id));
```
Maybe we could use `map` here to improve legibility

Reply:
yup, sure! done :+1: